components/mailgun/sources/new-click/new-click.js

```diff
@@ -0,0 +1,33 @@
+const common = require("../common-webhook");
+const { mailgun } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-click",
+  name: "New click",
+  description:
+    "Emit an event when the email recipient clicked on a link in the email. Open tracking must be enabled in the Mailgun control panel, and the CNAME record must be pointing to mailgun.org. See more at the Mailgun User's Manual [Tracking Messages](https://documentation.mailgun.com/en/latest/user_manual.html#tracking-messages) section",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    domain: { propDefinition: [mailgun, "domain"] },
+  },
+  methods: {
+    ...common.methods,
+    getEventName() {
+      return ["clicked"];
+    },
+    getEventType() {
+      return "CLICKED";
```

Comment:
Do you know if there's a reason Mailgun capitalizes only this event name, and not the others?

Reply:
I just inspected each webhook payload and noticed only `CLICKED` was sent capitalized.